webpack.config.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
       {test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader',
         'css-loader!postcss')},
       {test: /\.less$/, loader: ExtractTextPlugin.extract('style',
-        'css!postcss!less')}
+        'css!postcss!less')},
+      // handles the bootstrap glyphicons
+      {test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+        loader: 'url-loader?limit=100000&name=./statics/icons.svg'}
```

Comment:
Bootstrap imports glyphicons as woff, woff2, eot, ttf and svg and I need to somehow bundle those too. If I don't handle those files, for each I get an error like this:

![screen shot 2016-09-01 at 4 08 12 pm](https://cloud.githubusercontent.com/assets/20299984/18168539/e5f7f318-705e-11e6-8ead-868fc5735e7e.png)